nrf_security/Kconfig

```diff
@@ -350,99 +493,219 @@ config VANILLA_MBEDTLS_DHM_C
 	  MBEDTLS_DHM_C setting in mbed TLS config file.
 endmenu
 
+menu "ECC - Elliptic Curve Cryptography"
```

Comment:
I think changing this from menuconfig to menu is a step back in usability, as users must now enter the submenu to see if the group is configured or not.
The menuconfig allows user to completely disable ECC it is not needed.

Reply:
Fixed after some discussions